api/autoscaling/v2/horizontalpodautoscaler_webhook.go

```diff
@@ -64,6 +64,9 @@ var _ admission.CustomDefaulter = &HPAWebhook{}
 func (h *HPAWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	hpa := obj.(*v2.HorizontalPodAutoscaler)
 	tortoiseName, ok := hpa.GetAnnotations()[annotation.TortoiseNameAnnotation]
+	if !ok {
+		tortoiseName, ok = hpa.GetAnnotations()[annotation.TortoiseNameAnnotationV1]
+	}
```

Comment:
It should be the opposite: Try to get a value from the new annotation, and if not found, try to get from the old.